nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-resources/src/main/resources/conf/bootstrap.conf

```diff
@@ -63,6 +63,20 @@ nifi.bootstrap.sensitive.key=
 # HashiCorp Vault Sensitive Property Providers
 nifi.bootstrap.protection.hashicorp.vault.conf=./conf/bootstrap-hashicorp-vault.conf
 
+# Note: the following mapping properties only apply if a Sensitive Property Provider that uses property contexts
+# is configured.  Otherwise, these values are ignored.
+#
+# If no nifi.bootstrap.protection.xml.context.location.mapping.* properties are provided, the context for protected
+# properties uses their filename as a location prefix, e.g. "authorizers.xml||Manager Password".
```

Comment:
It looks like `getContextKey()` changed the separator from `||` to `/`.

Reply:
Good catch, thanks.